src/components/UI/Card/WishListCards/WishListCards.css

```diff
@@ -0,0 +1,24 @@
+
+.cart-card > .badge{
+    cursor: pointer;
+}
+
+.cart-card > .card-img {
+    height: 21em;
+}
+
+.cart-card{
+    height: fit-content;
+}
+
+.card-title span, h2{
+    font-size: 2rem;
+}  
+.cart-card:hover{
+    transform: scale(1.01);
+    opacity: 1.01;
+    box-shadow: 0 4px 2px rgba(0,0,0,0.02),
```

Comment:
Suggestion: you can add the hard-coded color values in variables.

Reply:
this is not a specific color but just shades of it
ok i will creata each as var